newm-server/gradlew.bat

```diff
@@ -0,0 +1,92 @@
+@rem

```

Comment:
remove

Reply:
Will remove it and add it to .gitignore file